src/codegen/generate.ts

```diff
@@ -360,11 +364,48 @@ export default class ApiGenerator {
 
   /**
    * Create a type alias for the schema referenced by the given ReferenceObject
+   * @param ignoreDiscriminator If true, the discriminator property of the schema referenced by
```

Comment:
When I see `@param` I expect [JSDoc](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) format. Would propose to move the comment inline just above `ignoreDiscriminator?: boolean,`